### PR TITLE
Fix `code` on `Challenge` which is incorrectly set as required

### PR DESCRIPTION
**Pull request by danerwilliams**
## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

#### PR Dependency Tree

* **PR #911** 👈

This tree was auto-generated by [Charcoal](https://github.com/danerwilliams/charcoal)

### CODE REVIEW

**Review by jbvolta — APPROVED** [review]

File: src/mfa/interfaces/challenge.interface.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export interface Challenge {
   createdAt: string;
   updatedAt: string;
   expiresAt?: string;
-  code: string;
+  code?: string;
```
Comment (amygdalama) [inline src/mfa/interfaces/challenge.interface.ts:7]: note that this interface is also shared w/ standalone MFA. does standalone have a required code?

Comment (amygdalama) [reply src/mfa/interfaces/challenge.interface.ts:7]: oh it was made optional

Comment (amygdalama) [reply src/mfa/interfaces/challenge.interface.ts:7]: 🤦‍♀️

Comment (amygdalama) [reply src/mfa/interfaces/challenge.interface.ts:7]: ignore me!!!

**Pull request merged**